### ISSUE TITLE
Reply content does not work

### ISSUE THREAD

**Issue by HansH** [pipacs/o2#114]
When using setReplyContent, this does not show up in the browser. For example, we can extend the facebookdemo by adding the following line to `FBDemo::FBDemo`

``
o2Facebook_->setReplyContent("You have succesfully authenticated to Facebook. ");
``

**Expected behaviour:**
After authenticating to Facebook, the browser displays the content "You have succesfully authenticated to Facebook."

**Actual behaviour:**
An empty page is shown. HTTP headers also do not seem to be received by the browser.

I tested this on MacOS High Sierra 10.13.3, using Firefox, Chromium and Safari. It also doesn't work when using a webview such as in the facebookexternalinterceptordemo.

Previously, I was on commit  c0e33a88c7a884268ab9d1b23729bbb6cafa9b6c. It did work back then, but after pulling in all changes since then, it stopped working. I don't know which change/changes caused this problem.

**Comment by sykyck** [comment]
Yes I was also testing the code and found out that O2replyserver uses Qtcpsocket for server shouldn't it use QsslSocket the byte array that is passsed to parsequeryparams function in o2replyserver  doesn't get the querytoken i believe its because the bytearray received is encrypted and needs to be decrypted with certificate

**Comment by niktischenko** [comment]
Should be fixed after #123 merge
